src/containers/Layout.tsx

```diff
@@ -12,17 +12,21 @@ const GeneralContainer = styled.main`
 interface LayoutProps {
   children?: React.ReactNode;
   title?: string;
+  errors?: Array<string>;
 }
 
-const Layout = ({ children, title }: LayoutProps) => {
+const Layout = ({ children, title, errors }: LayoutProps) => {
```

Comment:
after showed error need to clean it.
I opened our site.
Enter to page Bookmarks. And have error
![image](https://user-images.githubusercontent.com/103836064/193026099-f92fff41-b5e4-421b-b23f-598456a0994a.png)

it is not good.
But I continued. I logined and again visited bookmarks. I have again this message. But I had not any error in network. But this message left in redux and I all time see that error because I had not bookmarks and my request nothing back. 
![image](https://user-images.githubusercontent.com/103836064/193027136-173ff731-6576-417d-af71-f0412ec3f998.png)
![image](https://user-images.githubusercontent.com/103836064/193027537-8605beb0-1516-4939-b411-3c6c5c2af28e.png)